### PR TITLE
feat: documentation base page

### PR DESCRIPTION
**Pull request by mayuresh-tech**
Work done:
Landing page (Responsive)
Documentation page (Responsive)
Sidebar (Responsive)

To do:
Start working on alerts, avatar, and badges

Bugs:
Sidebar: 
       If the user tests responsive then the sidebar elements get invisible

### CODE REVIEW

**Comment by kushalsourav** [comment]
documentation is good

File: index.html [hunk 1]
```diff
@@ -69,7 +74,9 @@
         </nav>
     </header>
 
+    <!-- Section for landing page -->
     <section>
+        <!-- 4 X 4 Grid for displaying content -->
         <div id="grid-box">
             <div id="child">
```
Comment (deekshithmd) [inline index.html:81]: if possible use section instead of div

File: documentation/docs.html [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+            <!-- Mobile Section for Navigation Bar (Mobile view) -->
+            <div id="mobile-section">
+                <ul class="">
+                    <li class="logo-top">
+                        <div id="logo-container">
+                            <div id="logo">
+                                <img id="main-logo" width="30px" height="30px" src="favicon.svg" alt="Logo" />
+                                <div>Zippy UI</div>
+                            </div>
+                            <button id="cancel" class="no-bg cancel-btn">X</button>
```
Comment (deekshithmd) [inline documentation/docs.html:55]: you can use 'times;' instead of x

File: documentation/docs.html [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+                    </li>
+                    <li class="nav-link-box">
+                        Documentation
+                    </li>
+                </ul>
+            </div>
+
+            <!-- Social links on Navigation Bar -->
+            <div id="social-links">
+                <button class="no-bg hov"><i class="fa-brands fa-github"></i></button>
```
Comment (deekshithmd) [inline documentation/docs.html:69]: for small extent you can increase size of these icons

File: documentation/docs-css/style.css [hunk 4]
```diff
@@ -0,0 +1,137 @@
...
+}
+
+img {
+  max-inline-size: 100%;
+  block-size: auto;
+  object-fit: cover;
+  object-fit: top center;
+}
+
+:root {
```
Comment (deekshithmd) [inline documentation/docs-css/style.css:24]: I think using root at top is standard practice, check once

File: documentation/docs-css/style.css [hunk 5]
```diff
@@ -0,0 +1,137 @@
...
+    width: 100%;
+}
+
+#nav-section {
+  display: flex;
+  flex-flow: row wrap;
+  flex-grow: 1;
+}
+
+#nav-section-mobile,
```
Comment (deekshithmd) [inline documentation/docs-css/style.css:57]: use class if possible instead of id

**Review by harshkotkar5557 — APPROVED** [review]
looks to me. you can make utilizes for repeated properties.

**Pull request merged**